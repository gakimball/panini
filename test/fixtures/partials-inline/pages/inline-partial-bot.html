{{#*inline "layout-inline-partial-bot"}}
<p>layout-inline-partial-bot content</p>
{{/inline}}
<p>Page Body</p>