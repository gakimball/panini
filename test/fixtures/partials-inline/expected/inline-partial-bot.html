<html>
  <body>
    <p>No layout-inline-partial-top default content</p>
    <p>Page Body</p>

<p>layout-inline-partial-bot content</p>

  </body>
</html>
